sigmastate/src/main/scala/sigmastate/Values.scala

```diff
@@ -118,14 +146,89 @@ object Values {
     }
     def notSupportedError(v: SValue, opName: String) =
       throw new IllegalArgumentException(s"Method $opName is not supported for node $v")
+
+    import sigmastate.utxo.JitCostTable
+
+    def costOf(opName: String, opType: SFunc): Int = {
```

Comment:
Please do ScalaDoc for all the public methods

Reply:
methods removed